packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -448,6 +465,10 @@ export default baseMixins.extend<options>().extend({
 
       VInput.options.methods.onMouseUp.call(this, e)
     },
+    onObserve (entries: IntersectionObserverEntry[]) {
```

Comment:
This function receives 3 arguments, entries, observer and isIntersecting. You can simply use this.

Reply:
Will fix. I just look in v-lazy component (https://github.com/vuetifyjs/vuetify/blob/058b1fe857d3aab1dbdb2143a69e66b9435dc1e9/packages/vuetify/src/components/VLazy/VLazy.ts#L58), so, maybe it can be simplified too